src/promptflow/promptflow/_sdk/_service/app.py

```diff
@@ -51,6 +53,6 @@ def handle_exception(e):
                 error_info = e.message
             else:
                 error_info = str(e)
-            return jsonify({"error_message": f"Internal Server Error, {error_info}"}), 500
+            return {"message": "Internal Server Error", "error_message": error_info}, 500
```

Comment:
could you share an example of our error reponse and cloud PFS error response?

Reply:
cloud pfs error resposne:
![image](https://github.com/microsoft/promptflow/assets/17938940/dc30b181-5323-4a5a-a5b1-0cd84daea563)
local pfs error response:
![image](https://github.com/microsoft/promptflow/assets/17938940/472d0301-d922-40f8-ab72-46b95bbe27a5)